src/main/kotlin/com/hypto/iam/server/db/repositories/OrganizationRepo.kt

```diff
@@ -34,4 +34,13 @@ object OrganizationRepo : BaseRepo<OrganizationsRecord, Organizations, String>()
     suspend fun findById(id: String): Organizations? = dao().findById(id)
 
     suspend fun deleteById(id: String) = dao().deleteById(id)
+
+    suspend fun create(organization: Organizations) {
```

Comment:
I think a method to create by POJO is already present in BaseRepo or in DAOImpl. Can you please check

Reply:
Checked and removed